website/blog/2022-07-27-getting-started-with-the-dbt-semantic-layer.md

```diff
@@ -0,0 +1,165 @@
+
+**In the near Future with dbt Server and dbt Cloud proxy Server**
+
+When dbt Server and the dbt Cloud proxy Server release later this year, the flow of how metrics are consumed will change significantly. Your organization will no longer need to materialize each metric within a model in order to take advantage of the metric definition. Instead, you’ll be able to directly query the proxy server with the metric code provided and have the correct dataset returned to your BI tool of choice. 
+
+Additionally, integration partners will have built out experiences around Metrics using the Metadata API to create unique and creative ways for consumers to obtain metric data while abstracting away complexity. For example, a box that allows the user to select from a list of metrics, time grains, dimensions, and secondary calculation and then have the correct information returned to them regardless of the selection!
+
+This is an exciting future where data can be consumed in the context of how most people think about it - **metrics.**
+
+### So what is publicly available now?
+
+Right now, the two main open-source components that are publicly available are the `metric` node within `dbt-core` and the `dbt_metrics` package. Combined, these two can operate a simplistic semantic layer experience by allowing analytics engineers to define metrics and then use that metric with the package to materialize a table that contains the value for that metric on the defined date range. 
+
+It is limited in dimension flexibility (as the selected dimensions have to be defined at model creation) but is useful for those who want to ensure that metrics remain consistent across every BI tool. If you identify with any of the following conditions, you could be a good fit for implementing this as it exists today:
+
+- Your organization has at least a few key metrics
+- Your organization uses 1 or more BI tools
+- Your organization occasionally has issues around different metric calculations
+- You want to set your organization up for the dbt proxy server.
+- Your organization wants a centralized location for all metrics so everyone in the business knows where to look
+
+All of these are great reasons to begin exploring implementing metrics in your dbt project! If you’re curious about what an implementation of this might look like, we recommend referencing the [jaffle_shop_metrics](https://github.com/dbt-labs/jaffle_shop_metrics) repo!
+
+### What is still in development?
+
+Both the dbt Cloud proxy server and dbt Server are currently in development, with a scheduled release of later this year. If you’re curious about testing them once they are released, we recommend keeping an eye on our product announcements and then reaching out once they become publicly available! 
```

Comment:
Just triple checking we've confirmed this timeline with folks?

Reply:
Yep! Drew's blog post is going to mention releasing at Coalesce